server/src/test/java/org/eclipse/lsp/cobol/usecases/TestIdmsDcStatement.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2020 Broadcom.
```

Comment:
Due to the recent agreement, the year for the newly created files should be 2021